src/jaxsim/physics/algos/soft_contacts.py

```diff
@@ -247,11 +247,13 @@ def process_point_kinematics(
         return W_p_Ci, CW_vl_WCi
 
     # Process all the collidable points in parallel
-    W_p_Ci, CW_v_WC = jax.vmap(process_point_kinematics)(
-        model.gc.point.T, np.array(model.gc.body, dtype=int)
-    )
+    W_p_Ci, CW_vl_WC = jax.vmap(
+        lambda Li_p_C, parent_body: process_point_kinematics(
+            Li_p_C=Li_p_C, parent_body=parent_body
+        )
+    )(model.gc.point, jnp.array(model.gc.body))
```

Comment:
Is the lambda necessary?
```suggestion
    W_p_Ci, CW_vl_WC = jax.vmap(process_point_kinematics)(Li_p_C, parent_body)(
        model.gc.point, jnp.array(model.gc.body)
    )
```

Reply:
It is not necessary but PyCharm raises a warning (that I could ignore). I remove the lambda.

Reply:
Alright! Just for curiosity, which warning is raised?

Reply:
It's just PyCharm that fails sometimes to understand that the arguments of a generic `jax.vmap(my_fn)` are those of `my_fn`. Defining a lambda helps the IDE in this case. Not a big deal.

Reply:
For the records, the pattern using a lambda is quite delicate if it would be jax.jit instead of jax.vmap. In fact, a lambda created on-the-fly has a different `id` that re-triggers jit recompilations.